xml/net_proxy.xml

```diff
@@ -1465,12 +1455,12 @@ cachemgr_passwd disable all
 </screen>
 
   <para>
-   This puts the report in the directory of the Web server. Apache is required
+   This saves the report in the directory of the Web server. Apache is required
    to view the reports.
   </para>
  </sect1>
  <sect1 xml:id="sec-squid-moreinfo">
-  <title>More information</title>
+  <title>Further Information</title>
```

Comment:
Please revert, we have a standard name for this, https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-outline -> "More information"